cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CountryConversion.java

```diff
@@ -71,7 +75,32 @@ private CountryConversion() {
         }
     }
 
+    public static Optional<Country> fromIsoCode(String iso) {
+        if (iso == null) {
+            return Optional.empty();
+        }
+        try {
+            return Optional.of(Country.valueOf(iso.trim().toUpperCase()));
+        } catch (IllegalArgumentException ignored) {
+            // Ignore
+        }
+        return Optional.empty();
+    }
+
+    public static Country defaultCountry(AbstractIdentifiedObjectConversion c) {
+        if (LOG.isWarnEnabled()) {
+            LOG.warn("Default country converting {}", c.what());
+        }
+        return defaultCountry();
+    }
+
     public static Country defaultCountry() {
         return Country.values()[0];
     }
+
+    public static Country defaultCountryForBoundary() {
```

Comment:
You can delete this method and then, it is okay for me.

Reply:
Done